cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -903,6 +903,10 @@ func cognitiveAccountsByKind(
 	for resourceGroup, cogAccounts := range accountsByResourceGroup {
 		for _, cogAccount := range cogAccounts {
 			kindName := *cogAccount.Kind
+			// Replace "FormRecognizer" with "DocumentIntelligence"
+			if kindName == "FormRecognizer" {
+				kindName = "DocumentIntelligence"
```

Comment:
```suggestion
				kindName = "Document Intelligence"
```

Maybe having a space here will look a little better?